content/docs/command-reference/version.md

```diff
@@ -14,7 +14,7 @@ usage: dvc version [-h] [-q | -v]
 
 | Line                                        | Detail                                                                                                                                                                |
 | ------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash in case of a development version)                                                                                        |
+| [`DVC version`](#components-of-dvc-version) | Version of DVC (along with a Git commit hash if running a development version)                                                                                        |
```

Comment:
It looks like the output and fields have changed significantly- do we have a ticket for this?

Reply:
There should be one yeah, since this was updated in the core repo... ⌛

Reply:
#1635

Reply:
👍